contrib/monitored_resource_util/build.gradle

```diff
@@ -0,0 +1,17 @@
+description = 'OpenCensus Monitored Resource Util'
+
+apply plugin: 'java'
+
+[compileJava, compileTestJava].each() {
+    it.sourceCompatibility = 1.6
+    it.targetCompatibility = 1.6
+}
+
+dependencies {
+    compileOnly libraries.auto_value
+
+    compile libraries.google_cloud_core,
```

Comment:
We depend on this only for https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-core/src/main/java/com/google/cloud/MetadataConfig.java, I would consider to copy that for the moment.